docs/plans/PLAN_VIDEO.md

```
@@ -0,0 +1,170 @@
+# Поддержка видео в галереях
+
+  ## Summary
```

markdownlint-cli2 (0.22.1)

[warning] Headings must start at the beginning of the line

(MD023, heading-start-left)